### DISCUSSION TITLE
收益的来源

### DISCUSSION THREAD

**Discussion by pzjboy0924** [tearust/Chinese-wiki#8] [General]
# 第一、种植（Planting）
## （一）A种子
求补充
## （二）B种子
1．RA（Remote Attestation远程证明），是TEA项目的第一个也是最重要的“公共服务”，每个种植中的机器都会从中获得一些奖励，也可以叫做“公共奖励”。
2．AA（Availability Attestation），也是一种公共服务。
## （三）C种子
AA 同上
#  第二、TApp计费收入
当TApp开发者在创建TApp时选择的计费模式为每100个区块固定TEA支付时，那么托管该TApp的茶树（种植中的A或B种子）每100个区块都会获得固定金额的TEA，当然很少开发者会这么干，因为有可能面临破产。
#  第三、下注（Staking）
将解冻的A、B种子或1000个TEA下注到茶树（种植中的A或B种子）上去，然后获得分红，该茶树赚什么那么你也会分到什么（包括TEA和TApp token，按照staking比例分配）。
>  注意：插槽位置越低，获得的奖励百分比就越高，这意味着越早下注奖励越高，当然下注的人数过多，相应的奖励也会变低。
建议：种子的当前能力值还未达到峰值时去下注，挑那种插槽少的，等到你的种子能力值达到峰值后赎回来拿去种植。
# 第四、托管（Hosting）
使用自己的茶树去托管TApp，从而赚取TApp token红利。
>  注意：同一个茶树可以托管多个TApp，是由种子的当前能力值决定的。托管时须提供100个TEA的保证金，如果你的矿工离线超过1000个区块，那么你的保证金将被没收。如果该TApp没有收益，红利也就没有。
>  建议：有看好的TApp出来马上抢占host位置。
# 第五、投资 （Investment)
从bonding curve购买自己看好的TApp token， 越早投资收益越高。
# 第六、创建TApp（Create TApp）
开发好的应用，创建新的TApp，需要编程能力。
>  注意：在创建新的TApp时，有个初始token数量，可以自由选择，不同的数量需要不同数量的TEA余额。这个相当于自己投资自己的TApp，而且是第一个投资人。
# 第七、Theta 值
TApp 创建者决定的这个数值，意思为创建者将投资者购买TApp token多少比例的TEA用于TApp的开发，数值越大，创建人拿走越多，但是会降低投资人的热情。
# 第八、无论你是通过下注、托管、投资或者创建TApp获得的token，随着该token的增值，兑换到的TEA自然也会增加，反之则反。

**Comment by lxm218** [comment]
是每1000个区块支付一次
![image](https://user-images.githubusercontent.com/12605218/145620467-6ee6d247-d2d8-4a04-86b5-6791639e6dc1.png)

**Reply by kevingzhang** [reply]
测试网使用快速模式, 100区块就发奖励

**Comment by lxm218** [comment]
标题是收益来源，建议归纳为以下几点：
1. plant CML （矿工）
2. staking （矿工或投资人）
3. 创建TApp （开发者）- 开发者原始投资到bonding curve，theta值
4.  hosting （矿工）- TApp的两种计费模式 （固定支付TEA，或者支付Tapp token进行分红）
5. 投资TApp （开发者，矿工，投资人）- 买TApp token，获得收益分红
然后把你的内容分别装到这5点里面去

**Comment by lxm218** [comment]
AA - Availability Attestation； 类似朝阳群众的作用

**Reply by kevingzhang** [reply]
这是因为RA还没有开发完毕. AA又是目前唯一可以保证B矿机正常工作的监督方式. 所以先让AA顶着. 日后会逐步过渡到其他A.
